### DISCUSSION TITLE
Recommendation for large documents

### DISCUSSION THREAD

**Discussion by hanshupe007** [mindee/doctr#1722] [General]
We have large documents with more than 10000 pixels, and also A4 pages with 2144 x 1657. What's your experience and recommendation, as I understood docTr is internally rescaling it to 1024x1024. 

Should I crop large images into smaller pieces and process them separately, or could I fine tune a model on higher resolutions and larger documents?

**Comment by felixdittrich92** [comment]
Hi @hanshupe007 :wave:

Down scaling should not be a problem. For up scaling it looks more problematically. Do you have problems with your docs ?

**Comment by hanshupe007** [comment]
Using the original size of the large documents, the texts were too small to be detected, but cropping and zooming helped. I was just wondering if there is any recommendation or best practice regarding resolution, text size, etc
